Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.11 as base
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
+
+RUN source $HOME/.poetry/env && poetry update && poetry install
+
+FROM base as production
+ENV FLASK_ENV=production
+ENTRYPOINT ["gunicorn", "app:app"]
+CMD ["--config gunicorn.conf.py"]
+
+FROM base as development
+
+RUN source $HOME/.poetry/env && poetry install
+RUN source $HOME/.poetry/env && poetry config virtualenvs.create false --local
```

Comment:
You did poetry install in the base stage, so you don't need it here. 

And you should change config like `virtualenvs.create false` BEFORE you run poetry install.

Reply:
And if you shouldn't need to do the source poetry/env command each time. 

You might need to put something like this before you curl poetry, to add poetry to your PATH:
```
ENV POETRY_HOME=/poetry
ENV PATH=${POETRY_HOME}/bin:${PATH}
```

Reply:
Done